system-test/storage.ts

```diff
@@ -3442,6 +3442,35 @@ describe('storage', () => {
       assert.strictEqual(body, localFile.toString());
     });
 
+    it('should create a signed read url with usableFrom in the past', async () => {
+      const [signedReadUrl] = await file.getSignedUrl({
+        version: 'v4',
+        action: 'read',
+        usableFrom: Date.now() - 5000,
+        expires: Date.now() + 5000,
+      });
+
+      const res = await fetch(signedReadUrl);
+      const body = await res.text();
+      assert.strictEqual(body, localFile.toString());
+    });
+
+    it('should create a signed read url with usableFrom in the future', async () => {
+      const usableFromDate = new Date();
+      const usableFromMinutes = usableFromDate.getMinutes();
+      const expiresDate = new Date();
+      const expiresMinutes = expiresDate.getMinutes();
+      const [signedReadUrl] = await file.getSignedUrl({
+        version: 'v4',
+        action: 'read',
+        usableFrom: usableFromDate.setMinutes(usableFromMinutes + 60),
+        expires: expiresDate.setMinutes(expiresMinutes + 90),
+      });
+      const res = await fetch(signedReadUrl);
+      const body = await res.text();
```

Comment:
Instead of validating the text. Validate the res status code which is less brittle.